faqs/README.md

```diff
@@ -144,10 +144,12 @@ The following prices are exclusive of any local taxes and shipping/handling fees
 
 ### What do I get when I buy one?
 
-You get the Raspberry Pi board itself. The [official power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) and an SD card are not included, but they can be purchased at the same time from most places that sell the Raspberry Pi. You can also purchase pre-loaded SD cards; we recommend buying these from us or our licensed distributors rather than from third parties, as our software is updated regularly and cards sold by third parties can quickly become outdated.
+Raspberry Pi devices are available as just the board, or as part of a kit. When buying just the device, you get the Raspberry Pi board itself. The [official power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) and an SD card are not included, but they can be purchased at the same time from most places that sell the Raspberry Pi. You can also purchase pre-loaded SD cards; we recommend buying these from us or our licensed distributors rather than from third parties, as our software is updated regularly and cards sold by third parties can quickly become outdated.
 
 An official case for the Raspberry Pi is available separately [from our distributors](https://www.raspberrypi.org/products/). There are also lots of home-brew case discussions on the forum, as well as several third-party cases available. There are also loads of awesome [3D-printable cases](https://www.raspberrypi.org/blog/3d-printed-raspberry-pi-cases/) that community members have designed.
 
+Raspberry Pi (and other sellers) also sell our devices as part of kits, where you get everything you need to set up your Raspberry Pi and get it going. Our official kits include the [Raspberry Pi](https://www.raspberrypi.org/products/raspberry-pi-4-desktop-kit) itself, a case, keyboard, mouse, HDMI cable, power supply, SD card and beginners guide. In the case of the [Raspberry Pi 400](https://www.raspberrypi.org/products/raspberry-pi-400), the device itself is the case and keyboard, but the kit includes a mouse, HDMI cable, power supply, SD card and a book, The Official Raspberry Pi Beginners Guide.
```

Comment:
I've merged that in above. Maybe just this at the end of the answer:

`For full details of each product, see our [Product page](https://www.raspberrypi.org/products/).`